### PR TITLE
Upgrade dependencies and enable CI (GitHub Actions)

### PR DESCRIPTION
**Pull request by ggrossetie**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - '*'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version:
+          - 14.x
+          - 15.x
```
Comment (sapegin) [inline .github/workflows/ci.yml]: Something like this would be probably more useful: 

```suggestion
          - 12.x
          - 14.x
```

**Review by sapegin — COMMENTED** [review]
Thanks! A few comments:

1. We need semantic-release workflow too.
2. We'll need to remove the original Travis CI config file.
3. It's time to upgrade `engines` in `package.json` ;-)

I guess I can add that later since (1) requires tokens for publishing.

**Comment by ggrossetie** [comment]
> We need semantic-release workflow too.

I'm not familiar with `semantic-release`, should I run `npm run semantic-release` on every commit pushed on master/main branch?

> We'll need to remove the original Travis CI config file.

:+1: 

> It's time to upgrade engines in package.json ;-)

We should probably require Node >= 12. Using Node 14 (even if it's the current LTS) seems a bit aggressive. What do you think?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ggrossetie) [reply .github/workflows/ci.yml]: I'm fine with this change.
My reasoning was to use the current LTS (14) and the latest stable version (15) but I'm fine with 12.x and 14.x.

**Comment by ggrossetie** [comment]
Superseded by #3

**Pull request closed without merging**